content/developer/howtos/website_themes/theming.rst

```diff
@@ -0,0 +1,632 @@
+module and specifies the module's metadata. It must at least contain the `name` field, which is
+always required. It usually contains much more information.
+
+.. code-block:: python
+   :caption: ``/website_airproof/__manifest__.py``
+
+   {
+      'name': 'Airproof Theme',
+      'description': '...',
+      'category': 'Website/Theme',
+      'version': '15.0.0',
+      'author': '...',
+      'license': '...',
+      'depends': ['website'],
+      'data': [
+	     # ...
+      ],
+      'assets': {
+	     # ...
+      },
+   }
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 20 80
+
+   * - Field
+     - Description
+   * - name
+     - Human-readable name of the module (required)
+   * - description
+     - Extended description of the module, in `reStructuredText
+       <https://en.wikipedia.org/wiki/ReStructuredText>`_
+   * - category
+     - Classification category within Odoo
+   * - version
+     - Odoo version this module is addressing
+   * - author
+     - Name of the module author
+   * - license
+     - Distribution license for the module
+   * - depends
+     - Odoo modules which must be loaded before this one, either because this module uses features
+       they create or because it alters resources they define
+   * - data
+     - List of XML files
+   * - assets
+     - List of SCSS and JS files
+
+.. note::
+   To create a website theme, you only need to install the Website app. If you need other apps
+   (Blogs, Events, eCommerce, ...), you can also add them.
+
+Default options
+===============
+
+First, try to construct your theme by using Odoo's default options. This ensures two things:
+
+#. You do not re-invent something which already exists. For example, as Odoo provides an option to
+   add a border on the footer, you shouldn't recode it yourself. Instead, enable the default option
+   first, then extend it if needed.
+#. The user can still use all of Odoo's features with your theme. For example, if you recode the
+   border on the footer, you may break the default option or make it useless, giving the user a bad
+   experience. Also, your recode might not work as well as the default option, as other Odoo
+   features may rely on it.
+
+.. tip::
+   - Use 4 spaces per indentation level.
+   - Do not use tabs.
+   - Never mix spaces and tabs.
+
+.. seealso::
+   :doc:`Odoo coding guidelines <../../../contributing/development/coding_guidelines>`
+
+Odoo variables
+--------------
+
+Odoo declares many CSS rules, most entirely customizable by overriding the related SCSS variables.
+To do so, create a :file:`primary_variables.scss` file and add it to the `_assets_primary_variables`
+bundle.
+
+**Declaration**
+
+.. code-block:: python
+   :caption: ``/website_airproof/__manifest__.py``
+
+   'assets': {
+      'web._assets_primary_variables': [
+         ('prepend', 'website_airproof/static/src/scss/primary_variables.scss'),
```

Comment:
I don't have a clear idea of the difference, but shouldn't this be declared as `theme.ir.assets` instead? Is the way is implemented in your own themes: https://github.com/odoo/design-themes/blob/2d7c8ba5add9e00a9f9b9098ba2fa2a133cd2fb6/theme_common/data/ir_asset.xml#L428-L433